pyproject.toml

```diff
@@ -0,0 +1,53 @@
+[build-system]
+requires = ["setuptools>=61.0"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "dotenv-cli"
+description = "Simple dotenv CLI."
+authors = [
+    { name="Bastian Venthur", email="mail@venthur.de" }
+]
+license = { text = "MIT" }
```

Comment:
-> `license = { file="LICENSE" }` so it sources that information directly from the file